hr_planning/model/hr_planning_day_move_request.py

```diff
@@ -0,0 +1,139 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2014 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: David Coninckx <david@coninckx.com>
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+from openerp.osv import orm, fields
+from openerp.tools.translate import _
+from datetime import datetime
+from openerp.tools import DEFAULT_SERVER_DATE_FORMAT as DF
+from openerp.tools import DEFAULT_SERVER_DATETIME_FORMAT as DTF
+
+
+class hr_planning_day_move_request(orm.Model):
+    _name = "hr.planning.day.move.request"
+
+    def _employee_get(self, cr, uid, context=None):
+        ids = self.pool.get('hr.employee').search(
+            cr, uid, [('user_id', '=', uid)], context=context)
+        if ids:
+            return ids[0]
+        return False
+
+    _columns = {
+        'name': fields.char(
+            _('Name'), required=True,
+            states={'validate': [('readonly', True)]}),
+        'old_date': fields.date(
+            _('Old date'),
+            states={'validate': [('readonly', True)]}),
+        'new_date': fields.date(
+            _('New date'), required=True,
+            states={'validate': [('readonly', True)]}),
+        'hour_from': fields.float(
+            _('From'), states={'validate': [('readonly', True)]}),
+        'hour_to': fields.float(
+            _('To'), states={'validate': [('readonly', True)]}),
+        'employee_id': fields.many2one(
+            'hr.employee', 'Employee', required=True,
+            states={'validate': [('readonly', True)]}),
+        'state': fields.selection(
+            [('to_approve', 'To Approve'), ('validate', 'Approved')],
+            'Status', track_visibility="onchange", readonly=True),
+        'type': fields.selection(
+            [('add', 'Add'), ('move', 'Move')],
+            _('Type'), required=True,
+            states={'validate': [('readonly', True)]}),
+    }
+    _defaults = {
+        'state': 'to_approve',
+        'employee_id': _employee_get,
+    }
+
+    def create(self, cr, uid, vals, context=None):
+        if vals['type'] == 'move':
+            if (self._check_is_working(
+                    cr, uid, vals['employee_id'], vals['old_date'], context)):
+                return super(hr_planning_day_move_request, self).create(
+                    cr, uid, vals, context=context)
+            elif vals['old_date'] == vals['new_date']:
+                raise orm.except_orm('Warning',
+                                     _(u'You choose the same date'))

```

Comment:
Make this string translatable by using the _ function from openerp.tools.translate.
Also you could say 'You <b>chose</b> the same date'